en/introduction.md

```diff
@@ -21,6 +21,7 @@ Recommended practices are organized into two primary sections
 * The server hosting GTFS Realtime data should be reliable and consistently return validly-formatted protobuf-encoded responses. Fewer than 1% of responses should be invalid (protobuf errors or fetching errors).
 * The web-server hosting GTFS Realtime data should be configured to correctly report the file modification date (see HTTP/1.1 - Request for Comments 2616, under Section 14.29) so consumers can leverage the `If-Modified-Since` HTTP header. This saves producers and consumers bandwidth by avoiding transferring feed contents that haven't changed.
 * Feeds should provide protocol buffer-encoded feed content by default when queried via an HTTP request at the given URL - consumers should not need to define special HTTP accept headers to receive protocol-buffer encoded content.
```

Comment:
I think we should also adapt this statement to only refer to HTTP requests *without* a matching `Accept` header. Those clients that send `text/plain` by default as one of the accepted MIME types would (and should!) receive the textual representation.

```suggestion
* Feeds should provide protocol buffer-encoded feed content by default when queried via an HTTP request without an `Accept` header. Consumers should not need to send an HTTP `Accept` header to receive protocol-buffer encoded content.
```